lib/vdsm/storage/blockVolume.py

```diff
@@ -354,7 +354,6 @@ def calculate_volume_alloc_size(
                                                    initial_size)
 
         if preallocate == sc.SPARSE_VOL:
-            # Sparse qcow2
```

Comment:
No, this is always qcow2 - there is no raw-sparse on block storage.

Reply:
ok